src/planner/plan_builder.rs

```diff
@@ -67,4 +71,24 @@ impl PlanBuilder {
         let table_info = TableInfo::create(name, column_defs, constrains)?;
         Ok(PlanNode::CreateTable(CreateTablePlan { table_info }))
     }
+
+    fn sql_selection_to_plan(&self, selection: &ast::Expr) -> MySQLResult<PlanNode> {
+        let predicate = self.sql_expression_to_expr(selection)?;
+        Ok(PlanNode::Selection(SelectionPlan {
+            predicates: vec![predicate],
+        }))
+    }
+
+    fn sql_expression_to_expr(&self, expr: &ast::Expr) -> MySQLResult<Expr> {
+        match expr {
+            ast::Expr::BinaryOp { left, op, right } => match op {
+                ast::BinaryOperator::Eq => Ok(Expr::Equal {
```

Comment:
This never works without columnref

Reply:
@andylokandy Yes, but there isn't `SELECT` statement yet. So even if `ColumnRef` has been supported, this never works either.

Reply:
What is ColumnRef?